src/Generator.cpp

```diff
@@ -747,7 +747,7 @@ std::string halide_type_to_c_type(const Type &t) {
 
 int generate_filter_main_inner(int argc, char **argv, std::ostream &cerr) {
     const char kUsage[] =
-        "gengen\n"
+        "gengen \n"
```

Comment:
What's this doing here?

Reply:
Eep, I thought I had reverted that. It was a change I pushed to tickle the buildbots. Sorry.